README.md

```diff
@@ -268,6 +268,56 @@ Folowing command delete the session
 ```
 ocm backplane session --delete <session-name>
 ```
+## Backplane elevate
+If you need to run some oc command(s) with elevation using backplane-cluster-admin user, you can use the elevate command for this.
+
+The elevate command requires a none empty reason for the elevation. When a reason is provided it will be used for future usage, in order you do not have to provide a reason for each elevation commands. The reasons are stored in the kubeconfig context, so it is valid only for the cluster for which it has been provided. When a reason is created/used, the last used reason timestamp is updated in the context, and the reason will be kept for 20min after its last usage, in order to avoid bad usage.
```

Comment:
non-empty reason?

Reply:
non-empty means a non-empty string ;)

Reply:
Clarify this here: https://github.com/openshift/backplane-cli/blob/bb3749b074c1322e3393ac08ac81f7129b98e0e1/README.md?plain=1#L274